meetings.md

```diff
@@ -0,0 +1,65 @@
+# Meetings
+
+The purpose of this resource is to share details about how and when we
+meet.
+
+**Contents**
+- [Types](#types)
+  - [All Hands](#all-hands)
+    - [Standup](#standup)
+    - [Full](#full)
+  - [Holds](#holds)
+- [Scheduling](#scheduling)
+- [Hosting](#hosting)
+- [Expectations](#expectations)
+
+## Types
+
+### All Hands
```

Comment:
This structure seems confusing, can we just have "All-hands full meeting" and "All-hands standup" and reduce one layer of heading?

Reply:
ok, guess I was thinking this was just skeleton and that things common to both would emerge, but I see your point.

Reply:
Hm. Even now, there's stuff common to both, so I'm not sure agree with the suggestion. I don't any need to conserve heading levels I suppose?